### PR TITLE
react calculator component

### PR DESCRIPTION
**Pull request by soulemanou-software**
###
- I Created A folder called component with a js file in it as Calculator.js
- I fixed the linter errors
- I render the component to the apps.js

### CODE REVIEW

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @soulemanou-software,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly
✅ No linter error
✅ README is professional
✅ Good descriptive PR

## Required Changes ♻️
_Check the comments under the review._

I couldn't see file changes on the repo but below are the REQUIRED CHANGES to work on:

- [ ] Kindly make your `App.js` component a class-based component. It is required that you make all your components class-based and in a future milestone, you will refactor to hooks. 👍 
- [ ] Please remove the logo file you had when you created the app with create-react-app. You are asked to remove all files generated by create-react-app. Please work on that 👍  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] It would be nice if you always create a new branch for every major feature or milestone. For instance, in your previous milestone, the branch was set up, then the branch for this one would have been components / calculator-component. Kindly consider creating a new branch on your next milestone for every project 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
```
Comment (prolajumokeoni) [inline src/components/Calculator.jsx:1]: - [x] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

**Review by prolajumokeoni — APPROVED** [review]
Hi @soulemanou-software 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.jsx (hunk 1 above)
Comment (soulemanou-software) [reply src/components/Calculator.jsx:1]: thanks i will do that @prolajumokeoni

**Pull request merged**